### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by patrickbartlett**
Created user profile page that does not pull information from the backend. Edit button toggles typability of forms
Screen Shot 2021-03-03 at 2 14 41 AM
.

### CODE REVIEW

**Comment by patrickbartlett** [comment]
In like 12 of UserProfile.js, should I change the values to constants?

File: cloud-haven-react-app/src/components/UserProfile/UserProfile.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import './UserProfile.css'
+
+const UserProfile = () => {
+
+    const [ editing, setEditing ] = React.useState(false);
+    //                                                       '0' = open   null = closed
+    const [ accordionsOpen, setAccordionsOpen ] = React.useState([null, null, null]);
+
+    const toggleEditing = () => {
+        setEditing(editing ? false : true);
```
Comment (turrence) [inline cloud-haven-react-app/src/components/UserProfile/UserProfile.js:16]: could be abbreviated to !editing

Comment (patrickbartlett) [reply cloud-haven-react-app/src/components/UserProfile/UserProfile.js:16]: That's true, I copied the style of SignUp.js

**Review by glkidd — CHANGES_REQUESTED** [review]
I think that we should work on getting the user profile to match the Figma more before we merge this. Some things I notice: remove the borders and horizontal lines from the drop downs, labels and headers should be a blue color, edit button should be a little bigger, and the first name + last name fields should be lined up on the left and right with the drop downs.

**Review by Michelle5102 — CHANGES_REQUESTED** [review]
Left some comments on the code. Also, there's a `user-icon.png` file you'll probably want to delete since it seems empty.

Also, see below for the data the integration task force is thinking would be "user data"/"cloud haven data."

- First Name
- Last Name
- UID (user id)
- SSN (social security number)
- Birth Day
- Phone Number
- Email

File: cloud-haven-react-app/src/components/UserProfile/UserProfile.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import Card from 'react-bootstrap/Card'
+import Form from 'react-bootstrap/Form'
+import profile from "../../images/profile-user.png";
+import './UserProfile.css'
+
+const UserProfile = () => {
+
+    const [ editing, setEditing ] = React.useState(false);
+    //                                                       '0' = open   null = closed
+    const [ accordionsOpen, setAccordionsOpen ] = React.useState([null, null, null]);
```
[the remark this answers is not in the document]
Comment (patrickbartlett) [reply cloud-haven-react-app/src/components/UserProfile/UserProfile.js:13]: the defaultActiveKey attribute of the Accordion is open when set to '0', and i chose null as the value to designate as closed. I considered making constants of OPEN = '0' and CLOSED = null to make the code more readable

**Review by ShreyaTumma — CHANGES_REQUESTED** [review]
I agree with Gwennie, it would be nice if we could get the dropdowns to look more like the Figma prototype. I think it is also important for the first name and last name field to be in line with the dropdowns.

**Review by turrence — COMMENTED** [review]
On top of everyone else's comments, only thing I'd mention is that HTML tags aren't properly indented. Make sure to fix those up.

**Comment by ShreyaTumma** [comment]
Re-open this pull PR

**Pull request merged**